synchronization/src/cyface/java/de/cyface/synchronization/CyfaceSyncAdapter.java

```diff
@@ -114,7 +116,7 @@ private CyfaceSyncAdapter(final @NonNull Context context, final boolean autoInit
         this.rotationsUploadBatchSize = rotationsUploadBatchSize;
         this.directionsUploadBatchSize = directionsUploadBatchSize;
         progressListener = new HashSet<>();
-        addSyncProgressListener(new CyfaceSyncProgressListener(context));
+        addSyncProgressListener(new CyfaceConnectionListener(context));
```

Comment:
Since this is not called SyncProgressListener anymore, this method should get a new name as well.